corehq/apps/case_search/xpath_functions/query_functions.py

```diff
@@ -114,3 +114,11 @@ def match_all(node, context):
             serialize(node)
         )
     return filters.match_all()
+
+
+def or_(node, context):
+    from corehq.apps.case_search.filter_dsl import build_filter_from_ast
+    child_filters = [build_filter_from_ast(arg, context) for arg in node.args if arg]
+    if child_filters:
+        return filters.OR(child_filters)
+    return filters.match_none()
```

Comment:
Surely `OR('')` is equivalent to an empty query which is the same as `match all`?